src/patients/new/DuplicateNewPatientModal.tsx

```diff
@@ -0,0 +1,68 @@
+import { Modal, Alert } from '@hospitalrun/components'
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import useTranslator from '../../shared/hooks/useTranslator'
+import Patient from '../../shared/model/Patient'
+
+interface Props {
+  duplicatePatient?: Patient
+  show: boolean
+  toggle: () => void
+  onCloseButtonClick: () => void
+  onContinueButtonClick: () => void
+}
+
+const DuplicateNewPatientModal = (props: Props) => {
+  const { t } = useTranslator()
+  const { duplicatePatient, show, toggle, onCloseButtonClick, onContinueButtonClick } = props
+
+  const alertMessage =
+    'Patient with matching information found in database. Are you sure you want to create this patient?'
+
+  const body = (
+    <>
+      <Alert color="danger" title={t('Warning!')} message={t(alertMessage)} />
+      <div className="row">
+        <div className="col-md-12">
+          {`${t('Possible duplicate patient')}: `}
+          {duplicatePatient !== undefined &&
+            Object.entries(duplicatePatient).length === 1 &&
+            Object.entries(duplicatePatient).map(([key, patient]) => (
+              <Link key={key.toString()} to={`/patients/${patient.id}`}>
+                {patient.fullName}
+              </Link>
+            ))}
+          {duplicatePatient !== undefined &&
+            Object.entries(duplicatePatient).length > 1 &&
+            Object.entries(duplicatePatient).map(([key, patient]) => (
+              <li key={key.toString()}>
+                <Link to={`/patients/${patient.id}`}>{patient.fullName}</Link>
+              </li>
+            ))}
+        </div>
```

Comment:
I think that it is ok if it always displays a list no matter the number of duplicate patients returned. If we do that, then we can simplify this code.